purchase_control_menu/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* `Tecnativa <https://www.tecnativa.com>`__:
+
+  * David Vidal
```

Comment:
as all the effective code comes from odoo, I think we should : 
- set it as contributors
- keep the license to LGPL-3.

Don't you think ?

Reply:
Ok, I changed it in the manifest and readme regenerated